lib/rdkafka/admin.rb

```diff
@@ -605,6 +605,16 @@ def describe_acl(resource_type:, resource_name:, resource_pattern_type:, princip
       describe_acl_handle
     end
 
+    def oauthbearer_set_token(token:, lifetime_ms:, principal_name:, extensions: nil)
```

Comment:
do you want docstring on admin, consumer, producer version of this method?  Or just on the helper version?  Or everywhere?